File: Frontend/GWT/gwt-modular-dynamicform/src/com/nonagon/modular/dynamicform/client/template/Section.java
package com.nonagon.modular.dynamicform.client.template;

import com.google.gwt.core.client.JavaScriptObject;
import com.google.gwt.core.client.JsArray;

public class Section extends FormElementGroup {
	
	public static Section create() {
		
		Section instance = ((Section)JavaScriptObject.createObject());
		instance.init();
		
		return instance;
	}
	
	protected final native void init()/*-{
		
		this.__type = "Nonagon.Modular.DynamicForm.Template.Section, Nonagon.Modular.DynamicForm";
	}-*/;

	protected Section() {}
	
	public final native JsArray<FormElement> getChildren()/*-{ return this.Children; }-*/;
	public final native void setChildren(JsArray<FormElement> value)/*-{ this.Children = value; }-*/;
}

File: Backend/Nonagon.Modular.DynamicForm/Template/Section.cs
using System.Collections.Generic;

namespace Nonagon.Modular.DynamicForm.Template
{
	/// <summary>
	/// Form section.
	/// </summary>
	public class Section : FormElementGroup, IHasChildren
	{
		/// <summary>
		/// Gets or sets the list of <c>IFormElement</c>.
		/// </summary>
		/// <value>The children.</value>
		public IEnumerable<IFormElement> Children { get; set; }
	}
}

